web/templates/pages/list_web.php

```diff
@@ -296,9 +301,13 @@ class="units-table-row-action-link data-controls js-confirm-action"
 					</ul>
 				</div>
 				<div class="units-table-cell u-text-center-desktop">
-					<span class="u-hide-desktop u-text-bold"><?= _("IP Address") ?>:</span>
+					<span class="u-hide-desktop u-text-bold"><?= _("IPV4 Address") ?>:</span>
 					<?= empty($ips[$data[$key]["IP"]]["NAT"]) ? $data[$key]["IP"] : "{$ips[$data[$key]["IP"]]["NAT"]}" ?>
 				</div>
+				<div class="units-table-cell u-text-center-desktop">
+					<span class="u-hide-desktop u-text-bold"><?= _("IPV6 Address") ?>:</span>
+					<?=empty($ips[$data[$key]['IP6']]) ? "&nbsp;" : $data[$key]['IP6']; ?>
```

Comment:
No need for encoded space
```suggestion
					<?= $ips[$data[$key]['IP6']] ?? $data[$key]['IP6'] ?>
```

Reply:
May be in the actual revision it is so, but in the version before it was required. I tested it without this "special case" first and my tables were brocken. Therefore I realised it so with this "defined space". Otherwise this "cell" of the table was "spared".

Are you sure, that it will now work without this "defined space"?

Reply:
I know, that is the "old school". But I do not know other solution for it

Reply:
Correct, with the old tables this was required to prevent the row "collapsing". This is no longer an issue with the new tables, which this branch is using (old table style has been deleted in `main`)

Reply:
or is the solution this new space before the $ips?
I am not the big PHP expert if you use these short forms with ?? and so on.
Can you explain it please?

Reply:
ok, understood

Reply:
by the way:
`web/templates/pages/add_dns_rec.php:                                            <option value="">&nbsp;</option>
web/templates/pages/list_firewall.php:                                  <?php if (!empty($data[$key]['COMMENT'])) echo '' . $data[$key]['COMMENT']; else echo "&nbsp;"; ?>
web/templates/pages/list_server_info.php:       <div class="server-console-output">&nbsp;</div>
web/templates/pages/edit_dns_rec.php:                                           <option value="">&nbsp;</option>
web/templates/pages/list_web.php:                                       <?=empty($ips[$data[$key]['IP6']]) ? "&nbsp;" : $data[$key]['IP6']; ?>
web/templates/pages/list_services.php:                                          &nbsp;v<?= $sys["sysinfo"]["HESTIA"] ?>
`

Reply:
https://github.com/asmcc/hestiacp/commit/3a0c31fca93121f60c9f5cf1191c20293d28597d